blocks/small-manual-promo-block/features/small-manual-promo/default.jsx

```diff
@@ -1,23 +1,75 @@
 import React from "react";
 import PropTypes from "@arc-fusion/prop-types";
-import { useFusionContext } from "fusion:context";
-import { LazyLoad, isServerSide } from "@wpmedia/engine-theme-sdk";
-import { imageRatioCustomField } from "@wpmedia/resizer-image-block";
-import { SmallPromoPresentation } from "@wpmedia/shared-styles";
+import { useEditableContent } from "fusion:content";
+import { useComponentContext, useFusionContext } from "fusion:context";
+import {
+	formatURL,
+	Heading,
+	HeadingSection,
+	Image,
+	isServerSide,
+	Link,
+	MediaItem,
+	Grid,
+} from "@wpmedia/arc-themes-components";
+import { LazyLoad } from "@wpmedia/engine-theme-sdk";
 
-const SmallManualPromo = ({
-	customFields = { showImage: true, showHeadline: true, imageRatio: "3:2" },
-}) => {
+const BLOCK_CLASS_NAME = "b-small-manual-promo";
+
+const SmallManualPromo = ({ customFields }) => {
+	const { headline, imagePosition, imageURL, lazyLoad, linkURL, newTab, showHeadline, showImage } =
+		customFields;
+	const { registerSuccessEvent } = useComponentContext();
 	const { isAdmin } = useFusionContext();
-	const shouldLazyLoad = customFields?.lazyLoad && !isAdmin;
-	if (customFields.lazyLoad && isServerSide() && !isAdmin) {
-		// On Server
+	const shouldLazyLoad = lazyLoad && !isAdmin;
+
+	if (shouldLazyLoad && isServerSide()) {
 		return null;
 	}
 
+	const PromoImage = () => {
+		const { searchableField } = useEditableContent();
+
+		return showImage ? (
+			<MediaItem {...searchableField("imageURL")} suppressContentEditableWarning>
+				<Image alt={headline} src={imageURL} searchableField />
+			</MediaItem>
+		) : null;
+	};
+
+	const PromoHeading = () =>
+		showHeadline ? (
+			<Heading>
+				{linkURL ? (
+					<Link href={formatURL(linkURL)} openInNewTab={newTab} onClick={registerSuccessEvent}>
+						{headline}
+					</Link>
+				) : (
+					headline
+				)}
+			</Heading>
+		) : null;
+
 	return (
 		<LazyLoad enabled={shouldLazyLoad}>
-			<SmallPromoPresentation {...customFields} />
+			<HeadingSection>
+				<Grid
+					role="article"
+					className={`${BLOCK_CLASS_NAME} ${BLOCK_CLASS_NAME}__${imagePosition}`}
```

Comment:
Using BEM this is a modifier so should use `--` in the class

```suggestion
					className={`${BLOCK_CLASS_NAME} ${BLOCK_CLASS_NAME}--${imagePosition}`}
```

We would also need to update the Scss file to match this

Reply:
@matthewroach : Thanks! Changed in SCSS file as well.